CHANGELOG.md

```diff
@@ -55,6 +55,10 @@
   [SimplyDanny](https://github.com/SimplyDanny)
   [#5999](https://github.com/realm/SwiftLint/issues/5999)
 
+* Update `file_name` rule to consider `macro` types.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.